app/src/main/java/amazingme/database/FirebaseHelper.java

```diff
@@ -23,49 +32,28 @@ public static FirebaseUser getFirebaseUser() {
         return getFirebaseAuthInstance().getCurrentUser();
     }
 
-    public static void createNewUser(final String firstName,
-                                     final String lastName,
-                                     final String email,
+    public static void createNewUser(final String email,
```

Comment:
you said boolean didn't work for this because it's async?

Reply:
yeah, so the function returns before the async call gets back.